src/app/shared/components/maskedCopyableTextField.tsx

```diff
@@ -76,22 +77,24 @@ export class MaskedCopyableTextField extends React.Component<MaskedCopyableTextF
                         />
 
                         {allowMask &&
-                            <IconButton
-                                iconProps={hideContents ? { iconName: 'RedEye' } : { iconName: 'Hide' }}
-                                title={t(ResourceKeys.common.maskedCopyableTextField.toggleMask.label)}
-                                ariaLabel={t(ResourceKeys.common.maskedCopyableTextField.toggleMask.ariaLabel)}
-                                onClick={this.toggleDisplay}
-                            />
+                            <TooltipHost content={t(ResourceKeys.common.maskedCopyableTextField.toggleMask.label)}>
```

Comment:
Quick note- is 
`ResourceKeys.common.maskedCopyableTextField.toggleMask.label` different than `ResourceKeys.common.maskedCopyableTextField.toggleMask.ariaLabel` ?

If you check out [the Fabric example](https://developer.microsoft.com/en-us/fabric#/controls/web/tooltip) it looks like we can avoid having duplicate labels for the controls with tooltips by using the `aria-describedby` attribute:

```
<TooltipHost content="This is the tooltip" id={this._hostId}>
   <DefaultButton aria-describedby={this._hostId}>Hover Over Me</DefaultButton>
</TooltipHost>
```